### DISCUSSION TITLE
@route blade directive for checking current route

### DISCUSSION THREAD

**Discussion by bikify** [laravel/framework#40612] [Ideas] [answered]
It will be good for checking current route with @route directive

```php
@route('currentroute') active @endroute
```

**Comment by chuckrincon** [comment, marked as answer]
Hey there ✋🏻 .

Unfortunately there [was an attempt](https://github.com/laravel/framework/pull/34143) to introduce this blade directive, but it was rejected.

You could try using the current approach thought:
```blade
<input class="@if(request()->routeIs('route.name')) active @endif">
```
